server/src/main/java/io/druid/server/AsyncQueryForwardingServlet.java

```diff
@@ -266,6 +272,12 @@ protected void sendProxyRequest(
       }
     }
 
+    // Since we can't see the request object on the remote side, we can't check whether the remote side actually
+    // performed an authorization check here, so always set this to true for the proxy servlet.
+    // If the remote node failed to perform an authorization check, PreResponseAuthorizationCheckFilter
+    // will log that on the remote node.
+    clientRequest.setAttribute(AuthConfig.DRUID_AUTHORIZATION_CHECKED, true);
```

Comment:
have you tested this change? Does this mean request sent to broker will already have this attribute, but then SecuritySanityTestFilter on broker side will reject it ?

Reply:
yes, this path is tested as part of the integration tests, clientRequest is the original request received by the router, while proxyRequest is the request that's sent to the forwarding target

this is needed or clientRequest will fail the sanity check in PreResponseAuthorizationCheckFilter (which causes the integration tests to fail with a netty channel disconnect error)

Reply:
thanks, i thought it was the proxy request that was being sent to broker.